### ISSUE TITLE
Update manual to reflect new tech, Twink, Dashboard, API

### ISSUE THREAD

**Issue by alexgleason** [Uptwinkles/manual#8]
We basically do what Twink says. All chores can be reduced down to that. It's not open to interpretation anymore, we have a codebase that determines how things work. If you want to change how things work it's no longer a social problem to solve, it's a matter of sending a pull request to Twink's codebase.

**Comment by marykatefain** [comment]
Disagree. If we want to change the way things are done for anything non-trivial we still need to reach consensus, and submitting a PR should not be the only way to bring up ideas. People who have ideas but don't have the skills to implement them should feel free to express their ideas. Also people who don't have the time to implement something even if they have the skills should not feel like it will never be done bc they can't do it. And merging a PR does not require consensus, only one other person. This is very out of line with the spirit of the house. PRs for anything non-trivial should only be approved and merged when the group has already discussed and agreed upon something.

The manual should simply be updated to be more general, but still contain each individual rule. The only place this is really relevant is in the cleaning schedule section which can now say something more like "Each of the following chores is assigned to one person per week on a rotating basis".

The cooking schedule can also be much simplified since it still reference two people.

**Comment by alexgleason** [comment]
But the way to make change is still ultimately a matter of submitting a PR. People who can't do that can still make suggestions. If changes require consensus anyway then a programmer will agree to write new code.

**Comment by marykatefain** [comment]
Yes, I disagree with your statement: "If you want to change how things work it's no longer a social problem to solve, it's a matter of sending a pull request to Twink's codebase."

It's absolutely a social problem. It's a social problem FIRST. Once we figure out how we want it to look socially, /then/ we build the tech to match. Not the other way around. Also your statement very clearly implies that non-programmers can not change how things work.

**Comment by alexgleason** [comment]
Okay that's fair. All of software development is that way. I'm just excited how software can bring so much structure to it by self expressing what it is.
